src/http/representation/RepresentationMetadata.ts

```diff
@@ -304,6 +306,59 @@ export class RepresentationMetadata {
     return this;
   }
 
+  private setContentType(input?: ContentType | string): void {
+    // Make sure complete Content-Type RDF structure is gone
+    this.removeContentType();
+
+    if (!input) {
+      return;
+    }
+
+    if (typeof input === 'string') {
+      input = parseContentType(input);
+    }
+
+    for (const [ key, value ] of Object.entries(input.parameters)) {
+      const node = DataFactory.blankNode();
+      this.addQuad(this.id, SOLID_META.terms.contentTypeParameter, node);
+      this.addQuad(node, RDFS.terms.label, key);
+      this.addQuad(node, SOLID_META.terms.value, value);
+    }
+
+    // Set base content type string
+    this.set(CONTENT_TYPE_TERM, input.value);
+  }
+
+  /**
+   * Parse the internal RDF structure to retrieve the Record with ContentType Parameters.
+   * @returns An Record&lt;string,string&gt; object with Content-Type parameters and their values.
+   */
+  private getContentType(): ContentType | undefined {
+    const value = this.get(CONTENT_TYPE_TERM)?.value;
+    if (!value) {
+      return;
+    }
+    const params = this.getAll(SOLID_META.terms.contentTypeParameter);
+    return {
+      value,
+      parameters: Object.fromEntries(params.map((param): [string, string] => [
+        this.store.getObjects(param, RDFS.terms.label, null)[0].value,
+        this.store.getObjects(param, SOLID_META.terms.value, null)[0].value,
+      ])),
+    };
+  }
+
+  private removeContentType(): void {
+    this.removeAll(CONTENT_TYPE_TERM);
+    const params = this.store.getQuads(this.id, SOLID_META.terms.contentTypeParameter, null, null);
+    for (const quad of params) {
+      const labels = this.store.getQuads(quad.object, RDFS.terms.label, null, null);
+      const values = this.store.getQuads(quad.object, SOLID_META.terms.value, null, null);
+      this.store.removeQuads([ ...labels, ...values ]);
```

Comment:
```suggestion
      const paramEntries = this.quads(quad.object);
      this.store.removeQuads(paramEntries);
```
I think it is safe to just remove all references to those blank nodes as they should not be used anywhere else.